packages/files-ui/src/Components/Elements/NotificationsDropdown.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import { Button, BellIcon, MenuDropdown, Typography } from "@chainsafe/common-components"
+import { createStyles, ITheme, makeStyles } from "@chainsafe/common-theme"
+import dayjs from "dayjs"
+import relativeTime from "dayjs/plugin/relativeTime"
+dayjs.extend(relativeTime)
+
+const useStyles = makeStyles(({ palette, constants }: ITheme) =>
+  createStyles({
+    notificationsButton: {
+      position: "relative"
+    },
+    badge: {
+      position: "absolute",
+      background: palette.additional["red"][6],
+      color: palette.additional["gray"][1],
+      top: "-2px",
+      left: "13px",
+      borderRadius: constants.generalUnit,
+      padding: `${constants.generalUnit * 0.25}px ${constants.generalUnit * 0.5}px`,
+      fontSize: "11px",
+      lineHeight: "11px"
```

Comment:
we can achieve the pill look with a width of `1rem` and height of `.8rem` for instance
![image](https://user-images.githubusercontent.com/33178835/139664317-8de0392d-f83c-4b74-b96a-b0cc7fc61972.png)


```suggestion
      lineHeight: "11px",
      height: "0.8rem",
      minWidth: "1rem"
```